### ISSUE TITLE
Point.update of color not updated in legend (pie charts)

### ISSUE THREAD

**Issue by kzoon** [highcharts/highcharts#4917]
If you update the color of a point in a pie chart wit a legend, the pie color is updated OK, but the color of the legend items is not updated:

http://jsfiddle.net/k7r0s5km/5/

Only after hiding/showing by clicking the legend item does it get the new color...

**Comment by pawelfus** [comment]
Fixed on the bugfix branch: http://jsfiddle.net/k7r0s5km/6/

Issue fixed in #4622.

**Comment by kzoon** [comment]
Pawel,
Sorry that I missed that branch. 

I notice that the color of the connector to the data label is not updated (still yellow). Is that a separate issue?

**Comment by taalas** [comment]
Although this is closed I do have the same problem (connector color does not update). Is there a workaround for this?

**Comment by pawelfus** [comment]
Hi @taalas 

Let me report this as a separate issue: #7243

**Comment by taalas** [comment]
Thanks @pawelfus 

I wasn't sure whether it was ok to open a new issue for this. Thanks for taking the time!
